Snowflake.Data.Tests/IntegrationTests/SFPutGetTest.cs

```diff
@@ -560,6 +560,29 @@ public void TestPutGetGcsDownscopedCredential(
             }
         }
 
+        [Test]
+        public void TestPutFileWithSpaceAndSingleQuote()
+        {
+            var absolutePathPrefix = Path.Combine($"{Path.GetTempPath()}{Guid.NewGuid()} file path with space");
```

Comment:
It works because Path.GetTempPath() ends with a slash.
Maybe it could be changed to `Path.Combine(Path.GetTempPath(), $"{Guid.NewGuid()} file path with space");`

Reply:
Changed